apiserver/migrationtarget/migrationtarget_test.go

```diff
@@ -238,7 +238,7 @@ func (s *Suite) TestAdoptResources(c *gc.C) {
 	st := s.Factory.MakeModel(c, nil)
 	defer st.Close()
 
-	env := mockEnviron{Stub: &testing.Stub{}}
+	env := mockzEnviron{Stub: &testing.Stub{}}
```

Comment:
Doh, yup. Must have mashed the keyboard after running the tests but before committing. :( I should really rerun just before pushing to avoid that.